src/styles/partials/components/_page-header.scss

```diff
@@ -0,0 +1,46 @@
+.dg_page-header {
+	//background-image: url(/sebin/r/t/prototype-subpage-landing-header-bkg.jpg);
+	background-repeat: no-repeat;
+	background-size: cover;
+	background: $pale-blue;
+	color: $white;
+	padding: $padding-largest;
+	text-decoration: none;
+	p {
+		color: $white;
+		font-family: $default-body-font;
+		font-size: $h2-font-size;
+		font-style: italic;
+		padding-top: 25px;
+	}
+
+	.dg_page-header__box {
+		border-width: 10px;
+		display: flex;
```

Comment:
if we are already using the boostrap grid here, we can remove this class from the html and replace it with `row`.  See https://getbootstrap.com/docs/4.0/layout/grid/ for reference.

Reply:
done